packages/api/src/idpuser/service.test.ts

```diff
@@ -0,0 +1,39 @@
+import { fusionAuthClient } from "../fusionauth";
+import { getTwoFactorMethods } from "./service";
+
+jest.mock("../fusionauth");
+
+describe("/idpuser", () => {
+  const mockedFusionAuthClient = fusionAuthClient as jest.Mocked<
+    typeof fusionAuthClient
+  >;
+
+  afterEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  it("should correctly map values when there are two-factor methods", async () => {
```

Comment:
`stela` prefers `test` over `it`. Also there should probably be a test of what happens when `clientResponse.response.user.twoFactor.methods.length` is falsy.